projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -195,6 +202,11 @@ public static function delete_scheduled_update( $timestamp, $plugins ) {
 	 * Save the schedules for sync after cron option saving.
 	 */
 	public static function update_option_cron() {
+		// Do not update the option if the filter returns false.
+		if ( ! apply_filters( self::PLUGIN_CRON_SYNC_HOOK, true ) ) {
```

Comment:
@zaerl Let's add [hook docs](https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#4-hooks-actions-and-filters) to this for future reference.

How would you feel about using the hook string directly rather than abstracting it into a constant?
Feels to me like an opportunity to improve readability and debug-ability over DRY-ness.